packages/contracts-bedrock/test/L1/SystemConfig.t.sol

```diff
@@ -487,13 +487,46 @@ contract SystemConfig_Setters_Test is SystemConfig_Init {
     }
 }
 
+contract SystemConfig_Getters_Test is SystemConfig_Init {
+    /// @dev Tests that `superchainConfig()` returns the correct address.
+    function test_superchainConfig_succeeds() external view {
+        assertEq(address(systemConfig.superchainConfig()), address(superchainConfig));
+    }
+
+    /// @dev Tests that `guardian()` returns the correct address.
+    function test_guardian_succeeds() external view {
+        assertEq(systemConfig.guardian(), superchainConfig.guardian());
+    }
+
+    /// @dev Tests that `paused()` returns the correct value.
+    function test_paused_succeeds() external view {
+        assertEq(systemConfig.paused(), superchainConfig.paused(address(0)));
+    }
+
+    /// @dev Tests that `paused()` returns the correct value after pausing.
+
+    function test_paused_afterPause_succeeds() external {
```

Comment:
Add another test to check that `paused` is true when the ETHLockbox identifier is set